marathon-gradle-plugin/src/main/kotlin/com/malinskiy/marathon/MarathonRunTask.kt

```diff
@@ -38,17 +39,15 @@ private val log = MarathonLogging.logger {}
 open class MarathonRunTask : DefaultTask(), VerificationTask {
     @Input
     lateinit var flavorName: String
-    @Input
+    @Internal
     lateinit var applicationVariant: BaseVariant
```

Comment:
You should prefer `Property` to `lateinit`
Property connects with Gradle lifecycle and has greater API

Reply:
All fields are migrated.